### DISCUSSION TITLE
Custom element inside a table wont align correctly

### DISCUSSION THREAD

**Discussion by liusvel90** [quasarframework/quasar#11053] [General - Components / Directives / etc] [answered]
Hi everyone,
I'm trying to center a q-select inside a table (column calories), but I can't get it to work,

https://codepen.io/liusvel/pen/QWMKaYE?editors=1010

Any help I would appreciate

**Comment by metalsadman** [comment, marked as answer]
```html
...
<q-td...>
  <div class="row justify-center">
    <q-select ... />
  </div>
</q-td>
...

**Reply by liusvel90** [reply]
waaaoo so simple, thanks !!
